sdk/jdbc/azure-identity-providers-jdbc-mysql/README.md

```diff
@@ -0,0 +1,14 @@
+# Azure identity JDBC MySQL plugin library for Java
```

Comment:
this can't be empty, we should follow SDK convension and have the lib doc here

Reply:
Yes, I am working on the README.md, add this as template to pass the pipeline.